### ISSUE TITLE
Disable fuzzy matcher in intellisense

### ISSUE THREAD

**Issue by darkguy2008** [microsoft/vscode#128963]
I'm not sure if this is related to https://github.com/microsoft/vscode/issues/64367 since I'm using OmniSharp, but I haven't found anything in Google regarding this feature that I'd like to disable. 

There's different concepts of autocomplete in VSCode (fuzzy, intellisense, autocomplete, etc.) so I'm also puzzled to where exactly belongs what I mean, but here I go:

This is the **current** result, notice how **far** the scrollbar is from the suggestions box (plus, I don't know why it thinks it should start at "Process"...):
![image](https://user-images.githubusercontent.com/6304136/126092658-6fb3409a-6638-452d-88d7-0300fa1fe3a9.png)

This is the **expected** result:
![image](https://user-images.githubusercontent.com/6304136/126092719-2d661d86-035f-4567-8b13-92afbf457caf.png)

Is there any way to avoid this to happen? To **only** show the fields marked as properties (with the wrench icon) ?

I've already tried to set this in settings but it doesn't work:
![image](https://user-images.githubusercontent.com/6304136/126092773-4adc4cb7-42f1-4c50-8712-1cef760e2a20.png)

Thanks in advance!

**Comment by jrieken** [comment]
This is unlikely related to fuzzy matching your samples have no prefix. This is likely an issue with the default sorting that's provided by the extension. You can find that out yourself by hitting `cmd+/` while IntelliSense shows. It will toggle the details window into an explain view, showing score and sortText

**Comment by darkguy2008** [comment]
> This is unlikely related to fuzzy matching your samples have no prefix. This is likely an issue with the default sorting that's provided by the extension. You can find that out yourself by hitting `cmd+/` while IntelliSense shows. It will toggle the details window into an explain view, showing score and sortText

Thanks! The issue disappeared after a while but I managed to reproduce it again, here's what I see:

![image](https://user-images.githubusercontent.com/6304136/126358546-5fcc1f88-be84-4590-858f-ed76a45371f4.png)

And here's how it should have been:

![image](https://user-images.githubusercontent.com/6304136/126358605-16eead23-5a88-4990-88f8-ea556211b972.png)

I _think_ (but not sure) what I did to trigger it was:

1. Have a main A console project that includes a classlib project B
2. Create a new class in project B
3. Try to use that class in project A and try to use its initializer.

I'll be happy to provide more info if/when requested. In any case, I would still like to remove the items that have this icon:

![image](https://user-images.githubusercontent.com/6304136/126358931-24bbe4d5-2121-4b7c-ac98-9cf5de3e30ce.png)

Or in other words, **keep** only the ones with either the wrench or the purple/blue box (i.e. props/methods/vars) only.

Thanks!
